system/lvol.py

```diff
@@ -76,6 +81,12 @@
 # Create a logical volume of 512m.
 - lvol: vg=firefly lv=test size=512
 
+# Create a logical volume of 512m with disks /dev/sda and /dev/sdb
+- lvol: vg=firefly lv=test size=512 pvs=/dev/sda,/dev/sdb
```

Comment:
`pvs` param is `type='str'` but it should rather be `type='list'`. You will get a python list returned, but you can pass the values as shown above in the example
~~~
lvol: vg=firefly lv=test size=512 pvs=/dev/sda,/dev/sdb
~~~
but also as list
~~~
lvol: 
  vg: firefly
  lv: test
  size: 512
  pvs:
    - /dev/sda
    - /dev/sdb
~~~

Of course you will need to adjust the code to handle it as python list. e.g. Line 201 `pvs = ' '.join(pvs)`